mockup/patterns/tinymce/js/links.js

```diff
@@ -582,41 +600,42 @@ define([
         }
       }
     },
+
     guessImageLink: function(src) {
-      var self = this;
-      if (src.indexOf(self.options.prependToScalePart) !== -1) {
-        self.linkType = 'image';
-        self.$scale.val(self.tinypattern.getScaleFromUrl(src));
-        self.linkTypes.image.set(self.tinypattern.stripGeneratedUrl(src));
+      if (src.indexOf(this.options.prependToScalePart) !== -1) {
+        this.linkType = 'image';
+        this.$scale.val(this.tinypattern.getScaleFromUrl(src));
+        this.linkTypes.image.set(this.tinypattern.stripGeneratedUrl(src));
       } else {
-        self.linkType = 'externalImage';
-        self.linkTypes.externalImage.set(src);
+        this.linkType = 'externalImage';
+        this.linkTypes.externalImage.set(src);
       }
     },
+
     guessAnchorLink: function(href) {
-      var self = this;
```

Comment:
FWIW, I like the style of setting `var self = this;`. I know it's not necessary all the time but other times, it prevents confusing when using with inline functions.